.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: php
 php:
-  - 5.6
-  - 7.0
-  - 7.1
+  - 7.3
```

Comment:
we can also update the readme.md file with the PHP related info which are are no outdated